src/nvim/option.c

```diff
@@ -1804,19 +1804,20 @@ void set_init_1(void)
 # endif
     int len;
     garray_T ga;
-    bool mustfree;
 
     ga_init(&ga, 1, 100);
     for (size_t n = 0; n < ARRAY_SIZE(names); ++n) {
-      mustfree = FALSE;
+      bool mustfree = true;
```

Comment:
I think this is fine because an optimizing compiler will easily identify that `mustfree` is always true (except on unix). Though, using the `#ifndef` instead, I think you'd still need the `mustfree` anyway.